Podcast/SearchPeopleTableViewCell.swift

```diff
@@ -0,0 +1,108 @@
+                followButton.setTitleColor(.white, for: .normal)
+                followButton.setTitle("Following", for: .normal)
+            } else {
+                followButton.backgroundColor = .clear
+                followButton.setTitleColor(.podcastGreenBlue, for: .normal)
+                followButton.setTitle("Follow", for: .normal)
+            }
+        }
+    }
+    
+    
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        profilePictureImageView = UIImageView()
+        profilePictureImageView.layer.cornerRadius = imageViewHeight/2
+        profilePictureImageView.clipsToBounds = true
+        contentView.addSubview(profilePictureImageView)
+        
+        nameLabel = UILabel()
+        nameLabel.font = .systemFont(ofSize: 14, weight: UIFontWeightSemibold)
+        contentView.addSubview(nameLabel)
+        
+        detailLabel = UILabel()
+        detailLabel.font = .systemFont(ofSize: 12, weight: UIFontWeightRegular)
+        detailLabel.textColor = .podcastGrayDark
+        contentView.addSubview(detailLabel)
+        
+        followButton = UIButton()
+        followButton.layer.cornerRadius = 4
+        followButton.layer.borderColor = UIColor.podcastGreenBlue.cgColor
+        followButton.layer.borderWidth = 1
+        followButton.setTitle("Follow", for: .normal)
+        followButton.setTitleColor(.podcastGreenBlue, for: .normal)
+        followButton.titleLabel?.font = .systemFont(ofSize: 14, weight: UIFontWeightRegular)
+        followButton.addTarget(self, action: #selector(didPressFollowButton), for: .touchUpInside)
+        followButton.clipsToBounds = true
+        contentView.addSubview(followButton)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        profilePictureImageView.frame = CGRect(x: imageViewPaddingX, y: imageViewPaddingY, width: imageViewWidth, height: imageViewHeight)
+        let nameLabelX: CGFloat = profilePictureImageView.frame.maxX + imageViewLabelPadding
+        let followButtonX: CGFloat = frame.width - followButtonPaddingX - followButtonWidth
+        nameLabel.frame = CGRect(x: nameLabelX, y: nameLabelPaddingY, width: followButtonX - nameLabelX, height: nameLabelHeight)
+        detailLabel.frame = CGRect(x: nameLabelX, y: nameLabel.frame.maxY, width: nameLabel.frame.width, height: detailLabelHeight)
+        followButton.frame = CGRect(x: followButtonX, y: followButtonPaddingY, width: followButtonWidth, height: followButtonHeight)
+        separatorInset = UIEdgeInsets(top: 0, left: nameLabelX, bottom: 0, right: 0)
+    }
+    
+    func configure(for user: User) {
+        self.user = user
+        profilePictureImageView.image = getImage(fromOptional: user.imageURL)
```

Comment:
we don't need a getImage function, we're going to use an external library for async later, just use a dummy image url for now